java/manage-accounts.md

```diff
@@ -136,7 +119,7 @@ Credentials credentials = WalletUtils.loadBip39Credentials(password, mnemonic);
 
 ![](https://imgur.com/xN2Ruaj.png)
 
-By default, Web3j uses a derivation path equals to `m/44'/60'/0'/1`. However, it is possible to open another account on a different path:
+By default, Web3j uses a derivation path equal to `m/44'/60'/0'/1`. However, it is possible to open another account on a different path:
```

Comment:
@gjeanmart As a beginners post, this might need explanation.

Reply:
I added a link to [this article](https://medium.com/myetherwallet/hd-wallets-and-derivation-paths-explained-865a643c7bf2) from MEW which explain this complex but very important concept of Mnemonic derivation path!
Let me know if you think it is enough to explain and understand this.